src/types/UiaTextRangeBase.cpp

```diff
@@ -603,8 +604,17 @@ IFACEMETHODIMP UiaTextRangeBase::Move(_In_ TextUnit unit,
     // If we actually moved...
     if (*pRetVal != 0)
     {
-        // then just expand to get our _end
-        ExpandToEnclosingUnit(unit);
+        if (wasDegenerate)
+        {
+            // The range was degenerate before the move.

```

Comment:
```suggestion
            // GH#7342: The range was degenerate before the move.
```
We generally mark which bug this fixed as a paper trail.